configs/odac/is2re/eqv2_31M.yml

```diff
@@ -0,0 +1,77 @@
+includes:
+  - odac/configs/is2re/base.yml
+
+trainer: equiformerv2_energy
```

Comment:
Did you mean to upload this trainer as well? I'm only seeing a "equiformerv2_forces" in the repo.

Reply:
Uploaded now.